templates/previewDebugNotification.twig

```diff
@@ -5,7 +5,9 @@
     {% set mtmPreviewIdUrlStart = '' %}
     {% set mtmPreviewIdUrlEnd = '' %}
 {% endif %}
-
+<div piwik-alert="danger" id="debug-notification-alert" style="display: none">
+    <strong>Error!</strong> <span id="debug-notification-alert-message"></span>
```

Comment:
Any reason for not having `Error!` translatable?

Reply:
I missed it, added translation